.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,577 @@
+					button-primary: (
+						background-color: var(--primary-color),
+						color: var(--global-white),
+					),
+					button-primary-hover: (
+						background-color: var(--global-neutral-7),
+					),
+					button-primary-reverse: (
+						background-color: transparent,
+						border-color: var(--global-neutral-10),
+						color: var(--global-neutral-10),
+					),
+					button-primary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						color: var(--global-neutral-7),
+					),
+					button-secondary: (
+						background-color: var(--global-white),
+						color: var(--global-neutral-10),
+					),
+					button-secondary-hover: (
+						background-color: var(--global-neutral-7),
+						color: var(--global-white),
+					),
+					button-secondary-reverse: (
+						border-color: var(--global-white),
+						background-color: transparent,
+						color: var(--global-white),
+					),
+					button-secondary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						color: var(--global-neutral-7),
+					),
+					// DESIGN ASSUMPTION
+					// "Best match" assumptions were made w/ color and background
+					// color for the "default" this variant, based on both design in
+					// Zeplin as well as design trends in Figma. Please make any corrections.
+					button-default:
+						(
+							color: var(--global-neutral-10),
+							background-color: var(--global-white),
+						),
+					button-default-hover: (
+						background-color: var(--global-neutral-1),
+					),
+					// DESIGN ASSUMPTION: Date does not require component styling.
+					// DESIGN ASSUMPTION: Grid does not require component styling.
+					carousel:
+						(
+							--slides: 1,
+						),
+					carousel-actions: (
+						display: none,
+						padding-left: var(--global-spacing-3),
+						padding-right: var(--global-spacing-3),
+						//
+						// The width should place the action buttons near the edge of the carousel responsively
+						//
+						// width is derived from adding
+						//   --layout-content-max-width
+						//   2x button widths (--global-font-size-9)
+						//   2x button margins (--global-font-size-5)
+						// then dividing those by
+						//   --layout-max-width
+						// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+						//
+						width:
+							calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+					),
+					carousel-button: (
+						box-shadow: var(--global-box-shadow-1),
+						border-radius: var(--global-spacing-1),
+						background-color: var(--global-white),
+						height: var(--global-spacing-7),
+						width: var(--global-spacing-7),
+					),
+					carousel-track: (
+						gap: var(--global-spacing-2),
+						width: var(--layout-content-scale-width),
+					),
+					heading: (
+						font-weight: var(--global-font-weight-7),
+					),
+					icon: (
+						fill: var(--global-neutral-8),
+						height: var(--global-spacing-4),
+						width: var(--global-spacing-4),
+					),
+					input: (
```

Comment:
is there a form warning focus state? Looks like that #c986... orange is another color that isn't on the orange scale but exists as an alias token

![Screen Shot 2022-04-27 at 14 28 50](https://user-images.githubusercontent.com/5950956/165604945-20db6b43-95d7-40e4-aa6a-5c04cf274eef.png)

![Screen Shot 2022-04-27 at 14 29 46](https://user-images.githubusercontent.com/5950956/165605193-fecc791d-c525-493c-ba9c-42d371c105a9.png)

Reply:
@JackHowa : Thanks, good catch!
I added the tokens for warning and success with focus tokens. There aren't exact matches on the colors but I've put questions in the comments.

Reply:
**Update:** I've updated all the input colors with global color values confirmed by design.

Reply:
cool and design approves, it seems